client/db/types.go

```diff
@@ -382,6 +382,7 @@ func decodeBalance_v0(pushes [][]byte) (*asset.Balance, error) {
 // the zero-conf case, and balances for all asset-supporting exchanges. The
 // separate balances are required because how much is available depends on an
 // exchange's FundConf setting.
+// TODO: With no more FundConf setting, XC is no longer useful.
```

Comment:
@buck54321 Thoughts about how you'd like to rework Balance set and it's db serialization?